frontend/address-book/src/components/Nav.jsx

```diff
@@ -1,5 +1,60 @@
-const Nav = () => {
-  return <div>Nav</div>;
+import { Link } from "react-router-dom";
+import SearchBar from "./SearchBar";
+import API from "../utils/API";
+const Nav = (props) => {
+    const { userInfo } = props;
+    const wrap = {
```

Comment:
Not sure how React works, but shouldn't all of this css be in a different file so you can import it into other code for reuse?